components/raftstore/src/store/config.rs

```diff
@@ -725,34 +727,88 @@ impl Config {
     }
 }
 
-pub struct RaftstoreConfigManager(pub Arc<VersionTrack<Config>>);
+pub struct RaftstoreConfigManager {
+    scheduler: Scheduler<RefreshConfigTask>,
+    config: Arc<VersionTrack<Config>>,
+}
+
+impl RaftstoreConfigManager {
+    pub fn new(
+        scheduler: Scheduler<RefreshConfigTask>,
+        config: Arc<VersionTrack<Config>>,
+    ) -> RaftstoreConfigManager {
+        RaftstoreConfigManager { scheduler, config }
+    }
+}
 
 impl ConfigManager for RaftstoreConfigManager {
     fn dispatch(
         &mut self,
-        change: ConfigChange,
+        mut change: ConfigChange,
     ) -> std::result::Result<(), Box<dyn std::error::Error>> {
+        let change_str = format!("{:?}", change);
         {
             let change = change.clone();
-            self.0.update(move |cfg: &mut Config| cfg.update(change));
+            self.config
+                .update(move |cfg: &mut Config| cfg.update(change));
+        }
+        if let Some(ConfigValue::Module(mut apply_batch_system_change)) =
+            change.remove("apply_batch_system").map(Into::into)
+        {
+            if let Some(pool_size) = apply_batch_system_change.remove("pool_size") {
+                let pool_size: usize = pool_size.into();
+                CONFIG_RAFTSTORE_GAUGE
+                    .with_label_values(&["apply_pool_size"])
+                    .set(pool_size as f64);
```

Comment:
Could we move this to `write_change_into_metrics`?

Reply:
Fixed, thanks!